<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
margin : 0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>391</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(180, 232, 255);
font: 25 12pt &quot;Microsoft YaHei Light&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei Light'; font-size:12pt; font-weight:24; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Gulim'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="input_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 15px;                      /* 둥근 모서리 */
    border: 1px solid #4048e1;                /* 테두리 색상 (파란색) */
    background-color: rgb(46, 140, 255);
    color: rgb(255, 255, 255);                /* 텍스트 색상 (흰색) */
    font-size: 15px;                          /* 폰트 크기 */
</string>
    </property>
    <property name="text">
     <string>SQL 정렬</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>401</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(180, 232, 255);
font: 25 12pt &quot;Microsoft YaHei Light&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>188</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_title">
    <property name="geometry">
     <rect>
      <x>883</x>
      <y>183</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

    border-radius: 15px;                      /* 둥근 모서리 */
    border: 1px solid #4048e1;                /* 테두리 색상 (파란색) */
    background-color: rgb(253, 253, 253);
    color: rgb(0, 0, 0);                /* 텍스트 색상 (흰색) */
    font-size: 15px;                          /* 폰트 크기 */
</string>
    </property>
    <property name="text">
     <string>SQL 포맷</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>SQL 정렬 매크로</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(220, 0, 0);
font: italic 12pt &quot;Brush Script MT&quot;;
background-color: rgba(255, 255, 255, 0);  /* 투명 배경 */</string>
    </property>
    <property name="text">
     <string>Beta</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 12, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>1175</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 75 18pt &quot;Bahnschrift SemiCondensed&quot;;
</string>
     </property>
     <property name="text">
      <string>CORE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1110</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Alef&quot;;

color: rgb(255, 255, 255);
font: 75 18pt &quot;Bahnschrift SemiCondensed&quot;;</string>
     </property>
     <property name="text">
      <string>EASY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>1110</x>
       <y>40</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 75 18pt &quot;Bahnschrift SemiCondensed&quot;;</string>
     </property>
     <property name="text">
      <string>SOLUTIONS</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-top-right-radius: 40px;
border-bottom-right-radius: 40px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);
font: 24pt &quot;Cooper Black&quot;;</string>
      </property>
      <property name="text">
       <string>MOON </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="input_cnt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>740</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>글자수 : 0 </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_cnt">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>740</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>글자수 : 0</string>
    </property>
   </widget>
   <widget class="QToolButton" name="note">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>노트</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
